src/sp_repo_review/checks/general.py

```diff
@@ -64,10 +64,23 @@ class PY004(General):
 
     @staticmethod
     def check(package: Traversable) -> bool:
-        "Projects must have documentation in a folder called docs (disable if not applicable)"
+        "Projects must have documentation in a folder called doc or docs (disable if not applicable)"
         return len([p for p in package.iterdir() if "doc" in p.name]) > 0
 
 
+class PY004b(General):
```

Comment:
I don't think a check can end in a letter. I think this could be merged into the PY004 check? Though some projects have multiple `docs-*` folders, it's important not to error on those. Maybe just making sure `/doc` doesn't exist is an option?

Reply:
I think it works :-)

```
├── PY003 Has a LICENSE* file ✅
├── PY004 Has docs folder ✅
├── PY004b Documentation folder should be `docs` not `doc` ❌
│   Projects must have documentation in a folder called docs not doc
```

One of my main concern was to not mark a project as "not having docs", and think of a plan to convey to projects that `doc` used to be ok.

Reply:
Oh, interesting. I feel like I've encoded this somewhere. Maybe not.